catalogue/webapp/pages/works.tsx

```diff
@@ -318,6 +318,7 @@ const Works: NextPage<Props> = ({
                                 window.scrollTo(0, 0)
                               );
                             }}
+                            hideMobileTotalResults={true}
```

Comment:
Maybe I have misread this - but are we hiding total results on mobile in all cases?

Reply:
ahh nah we have two paginations which contain totalResults. We will hide the bottom total results for the mobile

Reply:
But we hide it on works and images?

Reply:
yes we will hide this on works and images for mobile.

Reply:
Ive added searchPrototype toggle to this to make sure it only displays only in the new search.